bin/run-versioned-tests.sh

```diff
@@ -25,11 +25,13 @@ fi
 
 export AGENT_PATH=`pwd`
 
-echo "${NPM7}"
+# Runner will default to CPU count if not specified.
+echo "JOBS = ${JOBS}"
+echo "NPM7 = ${NPM7}"
 
 if [[ "${NPM7}" = 1 ]];
 then
-  time ./node_modules/.bin/versioned-tests $VERSIONED_MODE -i 2 --all --strict --samples $SAMPLES ${directories[@]}
+  time ./node_modules/.bin/versioned-tests $VERSIONED_MODE -i 2 --all --strict --samples $SAMPLES --jobs $JOBS ${directories[@]}
```

Comment:
this breaks local versioned runs.  basically the first directory which is `test/versioned` is the jobs value so it's only running the versioned tests for external modules.

```
+ ./node_modules/.bin/versioned-tests --minor -i 2 --all --strict --samples 10 --jobs test/versioned/ test/versioned-external
```

```
 ✓ apollo-federation (5) 14s
 ✓ apollo-server (60) 2m
 ✓ apollo-server-express (50) 2m
 ✓ apollo-server-fastify (50) 2m
 • apollo-server-hapi/errors.test.js run 88 of 100): success
 ✓ apollo-server-koa (85) 2m
 ✓ apollo-server-lambda (50) 1m
 • koa-tests/koa-route.tap.js run 68 of 83): installing
 ✓ nextjs-tests (9) 1m
 ✓ superagent-tests (20) 26s
 • v2/amazon-dax-client.tap.js run 74 of 81): running
 • v3/s3.tap.js run 19 of 64): installing
```

Reply:
interesting. i didn't have that problem. but easy enough to tweak. you just may have to manually test for me.